WF-539: Migrate project structure (#12)
spatialos/FlexibleProjectExample: docs/changelog.md

# Changelog

* **18 Feb 2019:**

>   * Added a `build.sh` script to each worker type so that it is possible to build a single worker instance with the script.

Reviewer: ```suggestion
  * Added a `build.sh` script for each worker type so that it is possible to build different worker types individually.
```

  * Added a `download_dependencies.sh` script to `SpatialOS/scripts/` for downloading the schema compiler, the standard schema library, and the worker SDK in C# for MacOS, Windows, and Linux.

>   * Added a `generate-schema-descriptor.sh` script to `SpatialOS/scripts/` for generating a schema descriptor from the schemas. 

Reviewer: ```suggestion
  * Added a `generate-schema-descriptor.sh` script to `SpatialOS/scripts/` for generating a schema descriptor from the project schema. 
```

  * Added a `utils.sh` script to `SpatialOS/scripts` for storing paths to worker types, tools, and schemas.

>   * Added a `build-project.sh` script to `SpatialOS/scripts/`. It calls other scripts to build the project.

Reviewer: ```suggestion
  * Added a `build-project.sh` script to `SpatialOS/scripts/` for building the full project.
```

>   * Added a `build-project.sh` script to `SpatialOS/scripts/`. It calls other scripts to build the project.

Reviewer: Also don't forget to update the instructions on the top-level readme to point people to the new build-project.sh script.

Author: done
